srml/timestamp/src/lib.rs

```diff
@@ -326,6 +326,18 @@ impl<T: Trait> ProvideInherent for Module<T> {
 	}
 }
 
+impl<T: Trait> Time for Module<T> {
+	type Moment = T::Moment;
+
+	fn now() -> Self::Moment {
+		let now = Self::now();
+		// TODO TODO: we might change signature to return Option of Moment instead of this.
+		// or return 0.
+		assert!(!now.is_zero(), "Time hasn't been set yet");
```

Comment:
this ensure that we don't get a wrong value for now, but I'm mixed with the assert way of doing so

Reply:
`debug_assert` or comment it out - no panickers are allowed in runtime logic.